tools/gfx-unit-test/shader-cache-tests.cpp

```diff
@@ -689,6 +699,148 @@ namespace gfx_test
         }
     };
 
+    // Same as MultipleEntryShaderCache, but we now set the maximum entry count limit, so the cache
+    // should remove entries as needed when it reaches capacity.
+    struct CacheWithMaxEntryLimit : MultipleEntryShaderCache
+    {
+        // Check the correctness of the cache's entries by comparing the order of entries in the
+        // current state of the cache with what we expect.
+        void checkCacheFile(GfxIndex testIndex)
+        {
+            ComPtr<ISlangBlob> cacheContents;
+            cacheFileSystem->loadFile(shaderCache.cacheFilename, cacheContents.writeRef());
+
+            String apiString;
```

Comment:
Should move this into a util function. Or can just use `RenderApiUtil::getApiName()`.
The function should return `UnowendStringSlice` instead of `String` to avoid copy.

Reply:
Structure of this test has changed, see comment somewhere below